src/views/opm.tsx

```diff
@@ -81,18 +81,21 @@ const OPM: React.FC = () => {
 
   const DEFAULT = {
     duration: 10,
-    starttime: new Date(),
-    channel: CHANNELS.DESKTOP,
+    starttime: "",
+    channel: CHANNELS.ALL,
     promocode: "",
-    paymentType: "",
-    country: "US",
+    paymentType: PAYMENT_TYPES.ALL,
+    country: "",
   };
 
-  const [url, setUrl] = useState<string>();
+  const [url, setUrl] = useState<string>(
+    `${URL_OPM}?period=${DEFAULT.duration}&date=${DEFAULT.starttime}&channel=${DEFAULT.channel}&promocode=${DEFAULT.promocode}&paymentType=${DEFAULT.paymentType}&country=${DEFAULT.country}`,
+  );
```

Comment:
keep the initial state empty and set the URL in useEffect, otherwise API will get called with 2 different period in home page (since duration for home page and opm page charts are different)

Reply:
Resolved in latest commit.